hack/ccm/Makefile

```diff
@@ -0,0 +1,39 @@
+#TODO: Update the registry
+REGISTRY=quay.io/kabhat
+IMG=powervs-cloud-controller-manager
```

Comment:
registry and IMG needs to be changed to proper values, can be used from https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/64c9e1d17f1733c721f45a559edba3f4b712bcb0/Makefile#L42